addon/components/new-osf-navbar/template.hbs

```diff
@@ -0,0 +1,75 @@
+<div class="new-osf-navbar-styling">
+    <nav class="navbar navbar-inverse" id="navbarScope">
+        <div class="container">
+            <div class="navbar-header">
+                {{!Toggle navigation button for xs screens}}
+                <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".secondary-navigation" {{action 'closeSearch'}}>
+                    <span class="sr-only">Toggle navigation</span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                {{!OSF Brand}}
+                <a class="navbar-brand" href="/"><span class="osf-navbar-logo"></span> OSF </a>
+                {{!Primary navigation - toggle between OSF, Preprints, Registries, and Meetings}}
+                <ul class="primary-nav">
+                    <li class="dropdown">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" onclick={{action 'closeSecondaryAndSearch'}}><span class="caret"></span></a>
+                        <ul class="dropdown-menu service-dropdown" role="menu">
+                            {{#each osfServices as |service|}}
+                                <li><a href="{{service.url}}" onclick={{action "switchService" service.name}}>OSF<b>{{service.name}}</b></a></li>
```

Comment:
That spec does not have a space.  My guess is that it is supposed to resemble the branding on preprints and registries.